apps/prairielearn/python/zygote.py

```
@@ -510,7 +510,10 @@ def terminate_worker(_signum: int, _stack: types.FrameType | None) -> None:
                     # were running as the `executor` user
                     if drop_privileges:
                         # Kill all processes started by `executor`.
-                        os.system("pkill -u executor --signal SIGKILL")
+                        subprocess.run(
+                            ["pkill", "-u", "executor", "--signal", "SIGKILL"],
```

Ruff (0.15.14)

[error] Starting a process with a partial executable path

(S607)